semantic_release/dist.py

```diff
@@ -7,7 +7,7 @@
 
 def build_dists():
     commands = config.get('semantic_release', 'build_commands')
-    run(f'python setup.py {commands}')
+    run(f'{commands}')
```

Comment:
Let's remove the extra string interpolation.

```suggestion
    run(commands)
```